### ISSUE TITLE
Enhance UX/UI of newsletter subscription form.

### ISSUE THREAD

**Issue by Fupete** [italia/designers.italia.it#1029]
### Does a discussion already exist on the topic you are interested in, or on a similar topic?

- [X] I verified and it doesn't exist

### What

As subject: We need to improve the ux/ui of the newsletter signup form, now just an anchor to the footer form in the featured subheader.

### Why

It was just a temporary solution. Now it is the time to enhance it.

**Comment by iGieri** [comment]
What are you thinking about? Something like a new page with the form to subscribe to the newsletter?

**Comment by iGieri** [comment]
I like that idea. Are you already working on that?

**Comment by Fupete** [comment]
Not really @iGieri, do u want to contribute/help?

CC @bfabio @astagi

**Comment by iGieri** [comment]
Yes I'd like to contribute. I just need a bit of time to understand the code and then I can do a PR.

For any problems can I  text to you on this Issue?

**Comment by k-vijay-05** [comment]
@Fupete  Could you please assign it to me?
